components/formats-bsd/src/loci/formats/in/JPEGReader.java

```diff
@@ -207,8 +211,28 @@ protected void initFile(String id) throws FormatException, IOException {
         }
 
         HashMap<String, String> tags = exif.getTags();
-        for (String tagName : tags.keySet()) {
-          addGlobalMeta(tagName, tags.get(tagName));
+        for (String tagName : tags.keySet()) {  
+          if (tagName.equals(exif.getTagName(ExifSubIFDDirectory.TAG_X_RESOLUTION))) {
```

Comment:
Should we also parse the ResolutionUnit Exif tag to construct the the physical size using the correct unit?